tests/e2e/test_toplevel.py

```diff
@@ -253,3 +253,28 @@ def test_get_video(mp4_data):
     assert v2.description == v.description == desc
     assert v2.properties == v.properties == {}
     assert v2.labels == v.labels == ()
+
+
+def test_get_log_set():
+    name = f"logset-{uuid4()}"
+    desc = f"top-level test to create & get a log set {uuid4()}"
+    base_time = datetime.now(timezone.utc)
+    logs = [(base_time - timedelta(seconds=i), f"Log message {i}") for i in range(5)]
+
+    client = nm.get_default_client()
+
+    logset = client.create_log_set(name, logs, "absolute", desc)
```

Comment:
hmm this is not getting patched for me, and is trying to hit prod..

Reply:
I fixed this by having the client be a fixture, and provided the client to the default connection patch, as well as an arg to this test